app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if presenter.representative_id.present? && (fs = ::FileSet.load_instance_from_solr(presenter.representative_id)) %>
+<% if presenter.representative_id.present? && (fs = ::FileSet.find(presenter.representative_id)) %>
```

Comment:
:ok_hand: Can you drop a ticket # in here, and then I'll merge when the build is :green_apple: